<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CServerDlgBase</class>
 <widget class="QDialog" name="CServerDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/png/main/res/fronticonserver.png</normaloff>:/png/main/res/fronticonserver.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTreeWidget" name="lvwClients">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Client IP:Port</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Jitter Buffer Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Channels</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabServerSetup">
      <attribute name="title">
       <string>Server Setup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="lblDirectoryType">
           <property name="text">
            <string>Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxDirectoryType"/>
         </item>
         <item>
          <widget class="QLabel" name="lblRegSvrStatus">
           <property name="text">
            <string>STATUS</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="grbServerInfo">
         <property name="title">
          <string>My Server Info</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="lblServerName">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblLocationCity">
              <property name="text">
               <string>Location: City</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblLocationCountry">
              <property name="text">
               <string>Location: Region</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLineEdit" name="edtServerName"/>
            </item>
            <item>
             <widget class="QLineEdit" name="edtLocationCity"/>
            </item>
            <item>
             <widget class="QComboBox" name="cbxLocationCountry"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QCheckBox" name="chbJamRecorder">
           <property name="text">
            <string>Jam Recorder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtNewRecording">
           <property name="text">
            <string>New Recording</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRecorderStatus">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>STATUS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="lblCurrentSessionDir">
           <property name="text">
            <string>Session</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtCurrentSessionDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Chat Window Welcome (HTML/CSS Supported)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="tedWelcomeMessage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblLanguage">
           <property name="text">
            <string extracomment="Consider add &quot;(Lang)&quot; as a suffix to ensure that users who selected the wrong language find the correct button">Language</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CLanguageComboBox" name="cbxLanguage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pbtRecordingDir">
           <property name="text">
            <string>Recording Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtRecordingDir">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtClearRecordingDir"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="lblCustomDirectory">
           <property name="text">
            <string>Custom Directory address</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtCustomDirectory"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pbtServerListPersistence">
           <property name="text">
            <string>Server List Filename</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtServerListPersistence">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbtClearServerListPersistence"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chbStartOnOSStart">
         <property name="text">
          <string>Start Minimized on Windows Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chbDelayPanning">
         <property name="text">
          <string>Delay panning</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblUpdateCheck">
     <property name="text">
      <string>Update check</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLanguageComboBox</class>
   <extends>QComboBox</extends>
   <header>util.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lvwClients</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbxDirectoryType</tabstop>
  <tabstop>edtServerName</tabstop>
  <tabstop>edtLocationCity</tabstop>
  <tabstop>cbxLocationCountry</tabstop>
  <tabstop>chbJamRecorder</tabstop>
  <tabstop>edtCurrentSessionDir</tabstop>
  <tabstop>pbtNewRecording</tabstop>
  <tabstop>tedWelcomeMessage</tabstop>
  <tabstop>cbxLanguage</tabstop>
  <tabstop>pbtRecordingDir</tabstop>
  <tabstop>edtRecordingDir</tabstop>
  <tabstop>tbtClearRecordingDir</tabstop>
  <tabstop>edtCustomDirectory</tabstop>
  <tabstop>pbtServerListPersistence</tabstop>
  <tabstop>edtServerListPersistence</tabstop>
  <tabstop>tbtClearServerListPersistence</tabstop>
  <tabstop>chbStartOnOSStart</tabstop>
  <tabstop>chbDelayPanning</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
